bin/run-mochitests.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -e
+
+# Track each command when running on CI
+if [ $CI ]; then
+  set -x
+fi
+
+SCRIPT_DIR=$(dirname $0)
+
+. $SCRIPT_DIR/platform.sh
+
+# Folder where firefox binary is
+FIREFOX_DIR=$(dirname $FIREFOX_BIN)
+
+# Download firefox if not already done
+if [ ! -f $FIREFOX_BIN ]; then
+  $SCRIPT_DIR/fetch-firefox.sh
+fi
+
+# Folder where mochitest python app lives
+MOCHITEST_DIR=$ARTIFACTS_DIR/mochitest
+
+# Download artifacts files which are all the files necessary to run the test harnesses
+if [ ! -d $MOCHITEST_DIR ]; then
+  $SCRIPT_DIR/checkout-artifacts.sh
+fi
+
+# Create the browser folder runtests.py expects
+# it should contain browser-chrome.ini manifest
+# itself referencing other test manifest with relative paths
+# Use symlink to make that happen
+rm -rf $MOCHITEST_DIR/browser
+mkdir -p $MOCHITEST_DIR/browser
+ln -s $(realpath $SCRIPT_DIR/../client) $MOCHITEST_DIR/browser/
+ln -s $(realpath $SCRIPT_DIR/../shared) $MOCHITEST_DIR/browser/
+ln -s $(realpath $SCRIPT_DIR/../server) $MOCHITEST_DIR/browser/
+
+# Update the big manifest referencing all in-tree mochitest.ini or browser.ini files
+$SCRIPT_DIR/compute-ini.sh browser.ini $SCRIPT_DIR/../ > $MOCHITEST_DIR/browser/browser-chrome.ini
```

Comment:
We only run mochitest browser for now, we would need to make this script more generic to run the two other flavors. And tweak taskcluster to run the 3 flavors.